client/src/views/AdminDashboard.vue

```diff
@@ -65,6 +65,18 @@
         >
           T-Shirts
         </button>
+        <button
+          id="nav-photos-tab"
+          class="nav-link"
+          data-bs-toggle="tab"
+          data-bs-target="#nav-photos"
+          type="button"
+          role="tab"
+          aria-controls="nav-photos"
+          aria-selected="false"
+        >
+          Foto Download
```

Comment:
"Photos" as title would take up less space and still transfer enough information.